pom.xml

```diff
@@ -282,6 +283,18 @@
                 <version>${netty-tcnative-boringssl-static.version}</version>
                 <classifier>linux-x86_64</classifier>
             </dependency>
+            <!-- Ensure consistent commons-lang3 versions across dependencies and transitive dependencies -->
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
```

Comment:
I was running into that as well. 
Funny enough, we don't even have an explicit dependency on commons-lang3 anywhere

Reply:
I added an explicit commons-lang3 dependency to the server.